.github/workflows/check-publish.yml

```
@@ -0,0 +1,25 @@
+name: Check publish
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  check-publish:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+    runs-on: ${{ matrix.os }}
+    container:
+      image: paritytech/ci-unified:bullseye-1.70.0-2023-05-23-v20230706
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+
```

```suggestion
        - name: Rust Cache
           uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43 # v2.7.0
```

I'd suggest to use caching, so subsequent installations of `parity-publish` will take just few seconds instead of minutes

---

Thanks. Was looking to do this eventually but wasn't sure the prefered method.